src/src/PluginStructs/P145_data_struct.cpp

```diff
@@ -0,0 +1,598 @@
+// ----------------------------------------------------------------------------
+// P145 "Gases - MQxxx (MQ135 CO2, MQ3 Alcohol)"
+// Implementation of sensor abstraction
+// See _P145_MQxxx.ino
+// 2023 By flashmark
+// ----------------------------------------------------------------------------
+#include "../PluginStructs/P145_data_struct.h"
+
+#ifdef USES_P145
+
+#include "../Globals/ESPEasyWiFiEvent.h"   // Need to know when WiFi is ruining the ADC measurements
+#include "../Helpers/Hardware.h"           // Need to know the ADC properties
+
+/******************************************************************************/
+// Table with sensor type specific data
+// This table is stored in PROGMEM to save space in RAM
+// The data for the selected sensor type is copied to RAM, see P145_data_struct
+/******************************************************************************/
+const struct P145_SENSORDEF sensorData[] PROGMEM =
```

Comment:
Maybe the naming of this array isn't correct.
sensorData suggests "runtime" or "live" data.
But it actually is a sensor definition or some kind of template.
But given its type is called SENSORDEF, we should call the array `sensorDefs`